packages/parser-sample/src/grammar-demo.ts

```
@@ -0,0 +1,181 @@
+    identifier = [a-zA-Z_][a-zA-Z0-9_]*
+    number = [0-9]+ ("." [0-9]+)?
+    string = "\\"" [^\\"]* "\\""
+    quoted_string = "\\"" [^\\"]* "\\""
+    footer = "---" .*
+  }`,
+];
+
+for (const grammar of grammarExamples) {
+  console.log(`📖 Grammar Input:`);
+  console.log(grammar);
+  console.log();
```

Biome (1.9.4)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)